src/Akka.Persistence.Sql/persistence.conf

```diff
@@ -72,7 +72,7 @@
 
       # Only set to TRUE if unit tests pass with the connection string you intend to use!
       # This setting will go away once https://github.com/linq2db/linq2db/issues/2466 is resolved
-      use-clone-connection = false
+      use-clone-connection = true
```

Comment:
Change `use-clone-connection` to true.
The [original issue](https://github.com/linq2db/linq2db/issues/2466) was closed with a temporary patch, it is better to leave this to true.

Reply:
LGTM